src/dms/backend/EdFi.DataManagementService.Backend.Postgresql/PostgresqlAuthorizationRepository.cs

```diff
@@ -3,58 +3,46 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+using System.Text.Json;
 using EdFi.DataManagementService.Backend.Postgresql.Operation;
 using EdFi.DataManagementService.Core.External.Interface;
-using EdFi.DataManagementService.Core.External.Model;
 using Npgsql;
 
 namespace EdFi.DataManagementService.Backend.Postgresql;
 
 /// <summary>
 /// Implements IAuthorizationRepository to retrieve the authorization related data from the database.
 /// </summary>
-public class PostgresqlAuthorizationRepository(NpgsqlDataSource _dataSource) : IAuthorizationRepository
+public class PostgresqlAuthorizationRepository(NpgsqlDataSource _dataSource, ISqlAction sqlAction)
+    : IAuthorizationRepository
 {
-    public async Task<long[]> GetAncestorEducationOrganizationIds(
-        long[] educationOrganizationIds,
-        TraceId traceId
-    )
+    public async Task<long[]> GetAncestorEducationOrganizationIds(long[] educationOrganizationIds)
     {
         await using var connection = await _dataSource.OpenConnectionAsync();
```

Comment:
Are we ok with a new connection and transaction?

Reply:
Let's create a new ticket to reuse the original backend connection as a performance improvement.